.code-samples.meilisearch.yaml

```diff
@@ -114,6 +115,10 @@ update_settings_1: |-
             "a",
             "an"
         ],
+        "sortableAttributes": [
+          "price",
+          "author"
```

Comment:
```suggestion
          "title",
          "release_date"
```
For consistency, if this example is supposed to feat with the movies dataset I'm not sure there is `price` and `author` fields available

Reply:
Not sure it fits with the movie dataset since the index is `books` and not `movies`

Reply:
@alallema is right, very good catch! 

That's the example for the main settings route, which indeed uses the movies index. I changed it without thinking that it was not in the isolated context of sorting.